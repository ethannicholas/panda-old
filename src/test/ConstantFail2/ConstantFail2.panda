class Test {
    constant fail1 := new MutableString()

    @class
    method main() {
        constant fail2 := new Test()
    }
}