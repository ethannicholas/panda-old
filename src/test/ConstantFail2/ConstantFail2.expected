error: ConstantFail2.panda:2:5: constants must be immutable (found mutable type 'panda.core.MutableString')
error: ConstantFail2.panda:6:9: constants must be immutable (found mutable type 'Test'
2 errors
